gradle/jacoco.gradle

```diff
@@ -1,38 +1,9 @@
-apply plugin: 'jacoco'
+apply plugin: 'jacoco-android'
 
-project.afterEvaluate {
-  // Grab all build types and product flavors
-  def buildTypes = android.buildTypes.collect { type -> type.name }
-
-  buildTypes.each { buildTypeName ->
-    def sourceName, sourcePath
-    sourceName = sourcePath = "${buildTypeName}"
-    def testTaskName = "test${sourceName.capitalize()}UnitTest"
-    def coverageTaskName = "${testTaskName}Coverage"
-
-    // Create coverage task of form 'testFlavorTypeUnitTestCoverage' depending on 'testFlavorTypeUnitTest'
-    task "${coverageTaskName}"(type: JacocoReport, dependsOn: "$testTaskName") {
-      group = 'Reporting'
-      description = "Generate Jacoco coverage reports for the ${sourceName.capitalize()} build."
-
-      classDirectories = fileTree(
-              dir: "${project.buildDir}/intermediates/javac/${sourcePath}",
-              excludes: ['**/R.class',
-                         '**/R$*.class',
-                         '**/BuildConfig.*',
-                         '**/Manifest*.*'])
-
-      sourceDirectories = files(["src/main/java",
-                                 "src/$buildTypeName/java"])
-
-      executionData = files("${project.buildDir}/jacoco/${testTaskName}.exec")
-
-      reports {
-        xml.enabled = true
-        html.enabled = true
-        html.destination file("${buildDir}/jacocoHtml")
-      }
-    }
-    build.dependsOn "${coverageTaskName}"
-  }
+jacoco {
+  toolVersion = "0.8.3"
 }
+
+tasks.withType(Test) {
+  jacoco.includeNoLocationClasses = true
+}
```

Comment:
Do we need this 🤔? It's also included in https://github.com/mapbox/mapbox-navigation-android/blob/fad1cd4de295399b999dc5009b66cad2e567b01a/libandroid-navigation/build.gradle#L37-L39